packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -106,4 +136,14 @@ export const { command, description, builder, handler } =
   createYargsForComponentGeneration({
     componentName: 'cell',
     filesFn: files,
+    optionsObj: {
+      ...yargsDefaults,
+      list: {
```

Comment:
Another term instead of "list" might be "service" since really, the cell is generating the findMany service ... and whatever that is named.... plural model or maybe you have a Person model and `people` as your service so you would do

`yarn rw g cell person --list people`

or 

`yarn rw g cell person --service people`

or maybe

`yarn rw g cell person --useQuery people`